src/App.js

```diff
@@ -1,27 +1,64 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import {Route,Switch} from 'react-router-dom'
+import CountryDetail from './components/CountryDetail';
+import data from './data/countries.json'
 import './App.css';
 
+
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      countries: data
+    }
+  }
+
+
+
+
+
   render() {
+    const createClassString = () => {
```

Comment:
good seperation of concerns here, but maybe you could add this as a method to keep the render cleaner